encoding/basics-and-workflow.md

```diff
@@ -2,15 +2,245 @@
 
 ## Preparation
 
-downloading a source, looking at the video, some decisions
-(resolution(s) for the release, audio codec, group-specific
-requirements)
+It is important to have a good idea
+of what you want the result of the encode
+to be like.
+What will your video source be?
+What resolution should you encode it to?
+Do you use AAC or FLAC?
+8bit colors or 10bit colors?
+All of these are things
+that have to be accounted for
+and decided early on.
 
+Some groups have set standards.
```

Comment:
Except in germany. Because in germany having a standard set by a group implies that there is a group to be a part of to begin with.